packages/app/test/cypress/integration/20-basic-features/access-to-pagelist.spec.ts

```diff
@@ -90,6 +92,9 @@ context('Access to timeline', () => {
     cy.getByTestid('page-accessories-modal').parent().should('have.class','show').within(() => {
       cy.get('button.close').eq(1).click();
     });
-    cy.screenshot(`${ssPrefix}3-close-modal`, {capture: 'viewport'});
+    cy.screenshot(`${ssPrefix}3-close-modal`, {
+      capture: 'viewport',
+      blackout: ['[data-line="2"]:eq(0) > a > img', '[data-hide-in-vrt=true]']
```

Comment:
Did you need blackout here?
Do the `img` change before you open and after you close modal?